protocol/implementations/js/src/protocol-definitions.ts

```diff
@@ -39,17 +33,14 @@ export const aliceProtocolDefinition = {
           }
         ],
         // alice sends Orders, not receives them
-        Order: {
-          // whoever received the order that Alice sent in response to a Quote in response to an RFQ, can write back an OrderStatus to Alice.
-          OrderStatus: {
-            $actions: [
-              {
-                who : 'recipient',
-                of  : 'RFQ/Quote/Order',
-                can : 'write'
-              }
-            ]
-          }
+        OrderStatus: {
```

Comment:
fyi @KendallWeihe we're removing `Order`, so now it's `RFQ -> Quote -> OrderStatus`. i did remove the empty $actions arrays, so i'd appreciate another set of :eyes: here to make sure it will still work